jenkins/catapult/baseImageDockerfileGenerator.py

```diff
@@ -131,6 +131,9 @@ def rocks(self):
 		descriptor.options += DEPENDENCY_FLAGS['facebook_rocksdb']
 		descriptor.options += ['-DUSE_RTTI=1']
 
+		if self.compiler.c.startswith('gcc') and self.compiler.version == 12:
+			descriptor.options += ['-DFAIL_ON_WARNINGS=OFF']
```

Comment:
1. add comment about why we're (temporarily) disabling
2. can you just disable specific warnings (instead of all)

Reply:
1. ok
2. not sure it thats possible from cmake settings.